src/core/thread/network_data_tlvs.hpp

```diff
@@ -644,8 +598,7 @@ class BorderRouterEntry
 
 private:
     uint16_t mRloc;
-    uint8_t  mFlags;
-    uint8_t  mReserved;
+    uint16_t mFlags;
```

Comment:
The code above regarding `mFlags` is difficult to reasoning. Should we still use 2 bytes like below?
```suggestion
    uint8_t mFlagsHB;
    uint8_t mFlagsLB;
```